webrender/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "webrender"
-version = "0.17.0"
```

Comment:
There's a few other PRs in the queue that also update the version, so we can probably leave the version number change out of this (this drops the version number below what the queued PRs do).